README.md

```diff
@@ -2,7 +2,11 @@
 
 ## About
 
-Repository for the visualization of the [CC-Catalog](https://github.com/creativecommons/cccatalog) data.
+The landscape of licensed content is wide and varied. We have domains linking to other domains, different license types, and some metadata. This information is extracted from the Internet every month by Common Crawl and is available as the [CC-Catalog](https://github.com/creativecommons/cccatalog). It is fair to mention that we have 250 million works and growing! The goal of this project is to show the users how licensers are connected, their licensed content, and show how the licensing wave is expanding.
+
+A live demo of the project can be found in [2D](http://ec2-3-80-82-250.compute-1.amazonaws.com/) and in [3D](http://ec2-3-80-82-250.compute-1.amazonaws.com/visualization_3d.html)
```

Comment:
This is no longer accurate, the URL is http://dataviz.creativecommons.engineering/